lib/democracy.js

```diff
@@ -12,6 +12,9 @@ var util = require('util');
 var StringDecoder = require('string_decoder').StringDecoder;
 var decoder = new StringDecoder('utf8');
 var uuid = require('node-uuid');
+var AWS = require('aws-sdk');
+var Q = require("q");
```

Comment:
I think you don't have to use Q for promise. You can use just vanilla Promise. This is unnecessary additional dependency for this module.